Packs/PrismaCloud/Integrations/PrismaCloudV2/PrismaCloudV2.py

```diff
@@ -1474,10 +1490,14 @@ def config_search_command(client: Client, args: Dict[str, Any]) -> CommandResult
     search_id = args.get('search_id')
     sort_direction = args.get('sort_direction', 'desc')
     sort_field = args.get('sort_field', 'insertTs')
+    include_resource_json = args.get('include_resource_json', 'true')
+    include_additional_resource_fields = args.get('include_additional_resource_fields', 'false')
```

Comment:
```suggestion
    include_additional_resource_fields = args.get('include_additional_resource_fields', False)
```
Why not use boolean instead of string